DESCRIPTION

```diff
@@ -9,6 +9,7 @@ Authors@R: c(
       person("Hadley", "Wickham", role = c("ctb")),
       person("jQuery contributors", role = c("ctb", "cph"), comment = "jQuery library"),
       person("Leaflet contributors", role = c("ctb", "cph"), comment = "Leaflet library"),
+      person("Leaflet Providers contributors", role = c("ctb", "cph"), comment = "Leaflet Providers plugin"),
```

Comment:
For Leaflet, the contributors aren't copyright holders, according to the LICENSE file. (Though I suppose that if they haven't signed a CLA, it's possible the LICENSE file doesn't correctly reflect the copyright holders.)